core/lib/protobuf_config/src/proto/chain.proto

```diff
@@ -33,6 +33,7 @@ message StateKeeper {
   optional bool save_call_traces = 22; // required
   optional uint64 enum_index_migration_chunk_size = 26; // optional
   optional uint64 max_circuits_per_batch = 27; // required
+  optional uint64 miniblock_max_payload_size = 28; // required
```

Comment:
I don't remember if we already talked about this: have you already added to configs in all envs? If not, it cannot be required yet.

Reply:
Added here: https://github.com/matter-labs/gitops-kubernetes/pull/4871